spec/features/add_application_spec.rb

```diff
@@ -26,15 +28,10 @@
     create_user 'user'
     visit '/signin'
     signin_as 'user'
-    expect(page).to have_content('Welcome, user')
-    visit '/oauth/applications'
-    expect(page).to have_content('OAuth Applications')
-    create_new_application
-    expect(page).to have_content('Application created.')
-    expect(page).to have_content('Application: example')
-    expect(page).to have_content('Callback urls: https://localhost/')
-    expect(page.text).to match(/Application Id: [a-z0-9]+/)
-    expect(page.text).to match(/Secret: [a-z0-9]+/)
-    expect(page).to have_content('Trusted? No')
+    visit '/oauth/applications/new'
+
+    expect(page.status_code).to eq 500
   end
 end
+
+ENV["ACTION_DISPATCH_SHOW_EXCEPTIONS"] = "false"
```

Comment:
It's one way, I was hoping that at least it worked, but it didn't. So I have to do something else now anyway.